src/murfey/util/clem/images.py

```diff
@@ -5,123 +5,133 @@
 
 import logging
 from pathlib import Path
-from typing import List, Optional, Tuple
+from typing import Optional, Union
 
 import numpy as np
 from tifffile import imwrite
 
 # Create logger object to output messages with
 logger = logging.getLogger("murfey.util.clem.images")
 
+# For use by various functions in the script
+valid_dtypes = [
```

Comment:
Very minor but people would often use a tuple rather than a list here as the length doesn't change